xarray/backends/api.py

```diff
@@ -704,6 +704,8 @@ def open_mfdataset(paths, chunks=None, concat_dim='_not_supplied',
     parallel : bool, optional
         If True, the open and preprocess steps of this function will be
         performed in parallel using ``dask.delayed``. Default is False.
+    join : {'outer', 'inner', 'left', 'right', 'exact'}, optional
+        How to combine objects with different indexes.
```

Comment:
This seems a bit vague - should it at least direct the user towards the documentation for `align`?

Reply:
Updated docstrings